react/components/MiniCartFooter.js

```diff
@@ -0,0 +1,119 @@
+import React, { PureComponent, Fragment } from 'react'
+import classNames from 'classnames'
+import ProductPrice from 'vtex.store-components/ProductPrice'
+import { Button, Spinner } from 'vtex.styleguide'
+import { FormattedMessage } from 'react-intl'
+import PropTypes from 'prop-types'
+
+import { MiniCartPropTypes } from '../propTypes'
+import minicart from '../minicart.css'
+
+class MiniCartFooter extends PureComponent {
```

Comment:
If there isn't state, you could use a functional component

Reply:
Como todas as props são de tipos simples, quis aproveitar o pureComponent para evitar re-renders desnecessários. O que acham melhor?

Reply:
já que o render atualizou pra a versão mais nova, a gente pode usar o `React.pure` no componente funcional.

Reply:
updated to memo!